package.json

```
@@ -34,7 +34,7 @@
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-sourcemaps": "^0.6.3",
     "ts-node": "^8.10.2",
-    "typescript": "^3.9.10"
+    "typescript": "^4.9.3"
```

```suggestion
    "typescript": "~4.9.3"
```

TypeScript minor versions can include breaking changes, so we should unlock in on a specific minor version